### PR TITLE
Deprecate <MessageBanner> dismissable prop

### PR DESCRIPTION
**Pull request by ncovercash**
This prop has been replaced by `dismissible` and the code accounts for both spellings; we should document the mispelled one as deprecated.

### CODE REVIEW

**Review by bvsharp — APPROVED** [review]

File: lib/MessageBanner/readme.md [hunk 1]
```diff
@@ -72,7 +72,8 @@ aria-live | string | Sets the `aria-live`-attribute for the root element. | off,
 autoFocusDismissButton | bool | Autofocuses the dismiss button when the `<MessageBanner>` enters the DOM. This requires that the `dismissable`-prop is set to true | | false
 type | string | Sets the style of the `<MessageBanner>` | default, error, success, warning | default
 children | node | Renders the contents of the `<MessageBanner>` | |
-dismissable | boolean | Adds a close icon and makes the `<MessageBanner>` dismissable | true/false | false
+dismissable | boolean | Typo of dismissible, **deprecated** | true/false | false
+dismissible | boolean | Adds a close icon and makes the `<MessageBanner>` dismissable | true/false | false
```
Comment (zburke) [inline lib/MessageBanner/readme.md:76]: Petty: correct spelling in the description, too.

**Review by zburke — CHANGES_REQUESTED** [review]
Petty: 1 more typo.

File: lib/MessageBanner/readme.md (hunk 1 above)
Comment (ncovercash) [reply lib/MessageBanner/readme.md:76]: copy paste strikes again — doh

Comment (zburke) [reply lib/MessageBanner/readme.md]: In fact, `dismissable` shows up in other places too. Search for it, and catch 'em all please :)

**Comment by ncovercash** [comment]
> Petty: 1 more typo.

can't believe i did that lol

**Comment by ncovercash** [comment]
> Don't forget README line 72

fixed!

**Review by zburke — APPROVED** [review]

**Pull request merged**